spec/models/orchestration_stack/retirement_management_spec.rb

```diff
@@ -1,14 +1,15 @@
 describe "Service Retirement Management" do
+  let(:user) { FactoryGirl.create(:user_miq_request_approver, :userid => "admin") }
   before do
     @miq_server = EvmSpecHelper.local_miq_server
     @stack = FactoryGirl.create(:orchestration_stack)
   end
 
   it "#retirement_check" do
+    User.current_user = user
```

Comment:
@d-m-u 
Instead of using User.current_user you have to do something in a block
https://github.com/ManageIQ/manageiq/blob/master/spec/lib/rbac/filterer_spec.rb#L263